src/System.IO.FileSystem/tests/Performance/Perf.Directory.cs

```diff
@@ -63,7 +63,7 @@ public string GetTestDeepFilePath(int depth)
             StringBuilder sb = new StringBuilder(5000);
```

Comment:
You know the rough size of the output string, why not calculate this (at least roughly)? (Since you're already changing things...)